www-apps/kibana-bin/kibana-bin-6.2.4.ebuild

```diff
@@ -30,20 +30,20 @@ src_prepare() {
 	default
 
 	# remove bundled nodejs
-	rm -rv node || die
+	rm -r node || die
+
+	# remove empty unused directory
+	rm -r data || die
```

Comment:
If it's empty, use `rmdir`. That will trigger a failure when it stops being empty.